### PR TITLE
Support Bitbucket cloud oath

### PR DESCRIPTION
**Pull request by susan335**
Adds support for Bitbucket cloud authentication

### CODE REVIEW

**Comment by susan335** [comment]
Added support for bitbucket cloud OAuth authentication.

This is useful when using two-step verification.
This is optional.

If the environment variable is set, OAuth authentication will work.
* DANGER_BITBUCKETCLOUD_OAUTH_KEY
* DANGER_BITBUCKETCLOUD_OAUTH_SECRET

**Comment by susan335** [comment]
Generation of Key and Secret for OAuth Authentication

Open OAuth on the setting screen.
When you add a consumer key, it appears.

Screen Shot 2019-04-03 at 13 49 42

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 ## master
 
+## 6.0.3
+
+* Support bitbucket cloud OAuth [@susan335](https://github.com/susan335) #1115
```
Comment (RDIL) [inline CHANGELOG.md]:
```suggestion
* Support Bitbucket cloud OAuth [@susan335](https://github.com/susan335) #1115
```

**Comment by orta** [comment]
Alright, I found some time to add docs to this 👍

**Comment by orta** [comment]
Thanks @susan335 !

**Comment by orta** [comment]
merge on green

**Pull request merged**